packages/ckeditor5-utils/src/dom/emittermixin.ts

```diff
@@ -166,15 +180,21 @@ export default DomEmitterMixin;
  * @private
  */
 class ProxyEmitter {
+	private readonly _domNode: Node | Window;
+	private readonly _options: { capture: boolean, passive: boolean };
+
 	/**
-	 * @param {Node} node DOM Node that fires events.
+	 * @param {Node|Window} node DOM Node that fires events.
 	 * @param {Object} [options] Additional options.
 	 * @param {Boolean} [options.useCapture=false] Indicates that events of this type will be dispatched to the registered
 	 * listener before being dispatched to any EventTarget beneath it in the DOM tree.
 	 * @param {Boolean} [options.usePassive=false] Indicates that the function specified by listener will never call preventDefault()
 	 * and prevents blocking browser's main thread by this event handler.
 	 */
-	constructor( node, options ) {
+	constructor(
```

Comment:
Maybe we could utilize TypeScript access modifiers for constructors parameters so there is no need to redeclare and assign them in class?

Reply:
It would be difficult to find the best place to put JsDoc comment. Let's discuss that in #11723.